activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -304,9 +304,9 @@ module ClassMethods
       #   # creates avatar_attributes= and posts_attributes=
       #   accepts_nested_attributes_for :avatar, :posts, allow_destroy: true
       def accepts_nested_attributes_for(*attr_names)
-        options = { :allow_destroy => false, :update_only => false }
+        options = { :allow_destroy => false, :update_only => false, :index_errors => false }
         options.update(attr_names.extract_options!)
-        options.assert_valid_keys(:allow_destroy, :reject_if, :limit, :update_only)
+        options.assert_valid_keys(:allow_destroy, :reject_if, :limit, :update_only, :index_errors)
```

Comment:
I think we can use kwargs here and avoif the `extract_options!`, the `assert_valid_keys` and the default options altogether.